src/metabase/api/embed/common.clj

```diff
@@ -341,7 +341,8 @@
   :export?    false
   :audit      :never
   :type       :json
-  :default    default-eid-translation-counter)
+  :default    default-eid-translation-counter
+  :doc false)
```

Comment:
Added this, as I'm assuming we don't want to document this as an environment variable.